app/autolinker/server/settings.js

```diff
@@ -10,11 +10,18 @@ Meteor.startup(function() {
 
 	settings.add('AutoLinker', true, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, i18nLabel: 'Enabled' });
 
-	settings.add('AutoLinker_StripPrefix', false, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, i18nDescription: 'AutoLinker_StripPrefix_Description', enableQuery });
+	settings.add('AutoLinker_NewWindow', true, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, enableQuery });
 	settings.add('AutoLinker_Urls_Scheme', true, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, enableQuery });
 	settings.add('AutoLinker_Urls_www', true, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, enableQuery });
 	settings.add('AutoLinker_Urls_TLD', true, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, enableQuery });
-	settings.add('AutoLinker_UrlsRegExp', '(://|www\\.).+', { type: 'string', group: 'Message', section: 'AutoLinker', public: true, enableQuery });
 	settings.add('AutoLinker_Email', true, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, enableQuery });
 	settings.add('AutoLinker_Phone', true, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, i18nDescription: 'AutoLinker_Phone_Description', enableQuery });
+	settings.add('AutoLinker_StripPrefix', false, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, i18nDescription: 'AutoLinker_StripPrefix_Description', enableQuery });
+	settings.add('AutoLinker_StripPrefix_Scheme', true, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, enableQuery: [enableQuery, { _id: 'AutoLinker_StripPrefix', value: true }] });
+	settings.add('AutoLinker_StripPrefix_www', true, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, enableQuery: [enableQuery, { _id: 'AutoLinker_StripPrefix', value: true }] });
+	settings.add('AutoLinker_TrailingSlash', false, { type: 'boolean', group: 'Message', section: 'AutoLinker', public: true, enableQuery });
```

Comment:
The setting used in code was `AutoLinker_StripTrailingSlash` and not `AutoLinker_TrailingSlash`